modules/govuk_elasticsearch/spec/classes/govuk_elasticsearch_spec.rb

```diff
@@ -23,11 +23,11 @@
 
   describe '#manage_repo' do
     let(:params) {{
-      :version => '1.7.0',
+      :version => '1.4.3',
```

Comment:
Why is this needed?  Will this downgrade our cluster to using elasticsearch 1.4.3?